.drone.yml

```diff
@@ -1228,7 +1231,7 @@ steps:
           cat $WORKSPACE_DIR/curl_out.txt
           exit 1
         fi
-        curl $CREDENTIALS --fail -o /dev/null -X PUT "$RELEASES_HOST/releases/$product@$VERSION/assets/$(basename $file)"
+        curl $CREDENTIALS --fail -o /dev/null -X PUT "$RELEASES_HOST/releases/$product@$VERSION/assets/$(basename "$file" | sed 's/ /%20/g')"
```

Comment:
Is there a more general way to urlencode the filename?

Reply:
I couldn't find an easily available standalone utility to urlencode, aside from snippets like this https://stackoverflow.com/a/10660730/1264974

IMO it's fine for now, as our filenames are pretty uniform, and `Teleport Connect` is the first thing that needs to be urlencoded (all others roughly fall into the `[A-z0-9._-]` set)